Makefile

```diff
@@ -82,12 +78,8 @@ coverage-browse-report: coverage-html ## Open the HTML report in the browser
 bundle: ## Bundle the tutor package in a single "dist/tutor" executable
 	pyinstaller tutor.spec
 
-bootstrap-dev: ## Install dev requirements
-	pip install .
-	pip install -r requirements/dev.txt
-
-bootstrap-dev-plugins: bootstrap-dev ## Install dev requirements and all supported plugins
```

Comment:
any particular reasons for removing this?

Reply:
The `make` are not supposed to be part of the Tutor API, so I think it's OK to make breaking changes here. The goal of this change is to get rid of an unused command.